requirements.txt

```diff
@@ -54,7 +55,17 @@ notify2==0.3.1; sys_platform != 'win32' and sys_platform != 'darwin'
 # Uncomment line below or manually install after installing requirements.
 # pygobject>=3.10.2; sys_platform != 'win32' and sys_platform != 'darwin'
 
+# PyYAML does not have binary packages for Mac Users
+PyYAML --no-binary=PyYAML; sys_platform == 'darwin'
+PyYAML; sys_platform != 'darwin'
```

Comment:
Why not have non-binary for all?

Reply:
It's more efficient to use the binary when it's available. It's unfortunate that macos environment is impacted into a situation where they can't use it.